Panel/models/ad_model.go

```diff
@@ -4,12 +4,12 @@ import "gorm.io/gorm"
 
 type Ad struct {
 	gorm.Model
-	Title        string     `gorm:"type:varchar(255);not null"`
-	ImagePath    string     `gorm:"type:varchar(255);not null"`
-	BidValue     int        `gorm:"type:int;not null"`
-	IsActive     bool       `gorm:"type:boolean;not null"`
-	Clicks       int        `gorm:"type:int"`
-	Impressions  int        `gorm:"type:int"`
-	AdvertiserID int        `gorm:"type:int;not null"`
-	Advertiser   Advertiser `gorm:"foreignKey:AdvertiserID"`
+	Title        string `gorm:"type:varchar(255);not null"`
+	ImagePath    string `gorm:"type:varchar(255);not null"`
+	BidValue     int    `gorm:"type:int;not null"`
+	IsActive     bool   `gorm:"type:boolean;not null"`
+	Clicks       int    `gorm:"type:int"`
+	Impressions  int    `gorm:"type:int"`
+	RedirectLink string `gorm:"type:varchar(255);not null"`
+	AdvertiserID int    `gorm:"type:int;not null"`
```

Comment:
Retaining the original foreign key to Advertiser and removing AdvertiserID seems to be more compliant with database conventions.